vividus-plugin-mongodb/src/test/java/org/vividus/bdd/converter/StringToBsonConverterTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.converter;
+
+import java.util.stream.Stream;
+
+import org.bson.Document;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class StringToBsonConverterTests
+{
+    private static final String EMPTY_JSON = "{}";
+
+    private final StringToBsonConverter converter = new StringToBsonConverter();
+
+    static Stream<Arguments> documents()
+    {
+        return Stream.of(
+                Arguments.of(Document.parse(EMPTY_JSON), null),
+                Arguments.of(Document.parse(EMPTY_JSON), ""),
+                Arguments.of(Document.parse("{ \"id\" : 1 }"), "{\"id\":1}")
+            );
+    }
+
+    @MethodSource("documents")
```

Comment:
you could use csvsource here and it will be 1 Document.parse instead of 3

Reply:
under the hood it will be still 3 Document.parse

Reply:
from my pov MethodSource is more readable than
"{},", "{},''", "......"

Reply:
Less code - less support

Reply:
we should not prefer readability in favor of less code especially in unit tests